compiler/vm/core/src/shadowframes.c

```diff
@@ -1,6 +1,11 @@
 #include <robovm.h>
 
-void rvmPushShadowFrame(Env* env, ShadowFrame* frame) {
+#define LOG_TAG "shadowframes"
+
+void rvmPushShadowFrame(Env* env, void* functionAddress) {
+    ShadowFrame* frame = rvmAllocateMemory(env, sizeof(ShadowFrame));
```

Comment:
Yup my mistake, Clion took me to the test package where rvmAllocateMemory is implemented differently. 

Do you have this working? If so what are you running on, and can you share your sample?